src/ui/FunctionsOutlineView.cpp

```diff
@@ -364,15 +364,24 @@ FunctionsOutlineView::MessageReceived(BMessage* msg)
 			switch (code) {
 				case MSG_NOTIFY_EDITOR_SYMBOLS_UPDATED:
 				{
+					entry_ref newRef;
+					msg->FindRef("ref", &newRef);
+					Editor* editor = gMainWindow->TabManager()->SelectedEditor();
+					// Got a message from an unselected editor: ignore.
+					if (editor != nullptr && *editor->FileRef() != newRef) {
+						LogTrace("Outline view got a message from an unselected editor. Ignoring...");
+						return;
```

Comment:
Is this correct/needed? Looks like some code I introduced and then removed.

Reply:
I think is correct. I moved out from _UpdateDocumentSymbol.
It's needed because the notification is sent for every updates for all editors but we need to update the view only for tue selected editor.